apps/sap-commerce-cloud/app-actions/README.md

```diff
@@ -1 +1,13 @@
 # SAP App Actions
+
```

Comment:
Add a description here, something like "Hosted app actions are used to call the SAP APIs specifically for the private SAP Air app, so that the air header can be added to each network call."